src/main/app/src/components/common/LabelTooltip.tsx

```diff
@@ -0,0 +1,77 @@
+import { Backdrop, IconButton, makeStyles, Tooltip } from '@material-ui/core';
+import InfoOutlined from '@material-ui/icons/InfoOutlined';
+import React, { useState } from 'react';
+import { colors } from '#/src/colors';
+import CloseIcon from '@material-ui/icons/Close';
+import { useTranslation } from 'react-i18next';
+
+const useStyles = makeStyles(() => ({
+  backDrop: {
+    zIndex: 999,
+  },
+  closeIcon: { position: 'absolute', top: '5px', right: '5px' },
+  tooltip: {
+    backgroundColor: colors.white,
+    color: colors.black,
+    paddingLeft: '16px',
+    paddingRight: '35px', // Bigger to make space for close button
+  },
+  arrow: {
+    color: colors.white,
+  },
+}));
+
+type Props = { title: string };
+
+export const LabelTooltip = ({ title }: Props) => {
+  const classes = useStyles();
+  const { t } = useTranslation();
+
+  const [open, setOpen] = useState(false);
+  const handleClick = () => {
+    setOpen(!open);
```

Comment:
Ei taida tässä aiheuttaa ongelmaa, mutta näen näissä aina "stale state" riskin, etenkin jos tuota joku kehittää edelleen.
```suggestion
    setOpen(isOpen => !isOpen);
```